pushy/src/main/java/com/eatthepath/pushy/apns/ApnsChannelPool.java

```diff
@@ -152,10 +152,12 @@ private void acquireWithinEventExecutor(final Promise<Channel> acquirePromise) {
                         ApnsChannelPool.this.allChannels.add(channel);
                         ApnsChannelPool.this.metricsListener.handleConnectionAdded();
 
+                        log.debug("Created channel {}", channel);
                         acquirePromise.trySuccess(channel);
                     } else {
                         ApnsChannelPool.this.metricsListener.handleConnectionCreationFailed();
 
+                        log.debug("Failed to create channel", future.cause());
```

Comment:
Is this more important than debug?

Reply:
I think there's an argument to be made that this could be `warn` ("serious, but recoverable errors"). I think there's also an argument that it could be `debug` ("expected exceptions"). If this happens, callers will know because an attempt to send a push notification will fail with `future.cause()` as its cause; it's also reported in the metrics.

Let me think on it a bit.

Reply:
Yeah, let's call this `warn`.